webpack.conf.js

```diff
@@ -1,28 +1,71 @@
-var webpack = require('webpack');
+var CleanWebpackPlugin = require('clean-webpack-plugin'),
+    config = require('./config.json'),
+    HtmlWebpackPlugin = require('html-webpack-plugin'),
+    path = require('path'),
+    Visualizer = require('webpack-visualizer-plugin'),
+    webpack = require('webpack');
 
 module.exports = {
+    watch: false,
     devtool: 'source-map',
+    context: __dirname,
+    entry: {
+        main: './assets/js/app.js'
+    },
+    output: {
+        filename: '[chunkhash].theme-bundle.[name].js',
+        path: path.resolve(__dirname, "assets/dist"),
+        publicPath: 'assets/dist/', // TODO: cdnify - https://github.com/bigcommerce/paper/blob/master/index.js#L154
+    },
     bail: true,
     module: {
-        loaders: [
+        rules: [
             {
                 test: /\.js$/,
-                loader: 'babel',
                 include: /(assets\/js|assets\\js|stencil-utils)/,
-                query: {
-                    compact: false,
-                    cacheDirectory: true,
-                    presets: ['es2015-loose'],
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        compact: false,
+                        cacheDirectory: true,
+                        presets: [['es2015', {loose: true, modules: false}]],
```

Comment:
loose mode was for ie10:  https://github.com/bigcommerce/cornerstone/commit/f6f2ec08cd026b478654eedd721565e78d0f9ced